From 091eaf875d684b5db795c1f278455624c22fc7b7 Mon Sep 17 00:00:00 2001
From: Roy Marples <roy@marples.name>
Date: Fri, 10 Mar 2017 14:44:26 +0000
Subject: [PATCH] Don't crash when calling IPv4LL without an address in state

---
 ipv4ll.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ipv4ll.c b/ipv4ll.c
index fd37ac8d..0877ce73 100644
--- a/ipv4ll.c
+++ b/ipv4ll.c
@@ -144,7 +144,7 @@ ipv4ll_env(char **env, const char *prefix, const struct interface *ifp)
 	struct in_addr netnum;
 
 	assert(ifp != NULL);
-	if ((state = IPV4LL_CSTATE(ifp)) == NULL)
+	if ((state = IPV4LL_CSTATE(ifp)) == NULL || state->addr == NULL)
 		return 0;
 
 	if (env == NULL)
